patches/tModLoader/Terraria/Player.TML.cs

```diff
@@ -119,9 +119,9 @@ public partial class Player {
 		public bool InModBiome(ModBiome baseInstance)=> modBiomeFlags[baseInstance.ZeroIndexType];
 
 		/// <summary>
-		/// The zone field storing if the player is in the purity/forest biome. Updated in <see cref="UpdateBiomes"/>
+		/// The zone property storing if the player is in the purity/forest biome. Updated in <see cref="UpdateBiomes"/>
 		/// </summary>
-		public bool zonePurity { get; set; } = false;
+		public bool ZonePurity { get; set; } = false;
```

Comment:
Why not ZoneForest? Also, unnecessary assignment.
I think that the biome PR should've eliminated all the Zone properties, not add new ones. There should  be wrapping over vanilla, same way as with DamageClass.